### DISCUSSION TITLE
不同字体大小的一串字符中小字号的字符未被识别出来

### DISCUSSION THREAD

**Discussion by XiaoleiJia1005** [PaddlePaddle/PaddleOCR#12604] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：

   **MacOS 13.5**

- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：

  paddleocr :                2.7.0.0                 pypi_0    pypi
  paddlepaddle :        2.5.1                    pypi_0    pypi

- 运行指令/Command Code：

   ```
   from paddleocr import PaddleOCR
   ocr = PaddleOCR(use_angle_cls=False, lang='ch', show_log = False)
   res = ocr.ocr("testdata/missing.PNG", cls=False, det=False)
   print(res)
   ```
- 完整报错/Complete Error Message：
   ```
   [[('2中', 0.9981159567832947)]]
   ```
我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no): no

请尽量不要包含图片在问题中/Please try to not include the image in the issue.
![missing](https://github.com/PaddlePaddle/PaddleOCR/assets/8534661/830e8e40-1c61-4e35-ae4e-1058f67c4741)

**Comment by MarginGitHub** [comment]
增加此类样本，同时减少下采样

**Comment by CapitaineNemo** [comment]
> 请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem
> 
> * 系统环境/System Environment：
>   **MacOS 13.5**
> * 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
>   paddleocr :                2.7.0.0                 pypi_0    pypi
>   paddlepaddle :        2.5.1                    pypi_0    pypi
> * 运行指令/Command Code：
>   ```
>   from paddleocr import PaddleOCR
>   ocr = PaddleOCR(use_angle_cls=False, lang='ch', show_log = False)
>   res = ocr.ocr("testdata/missing.PNG", cls=False, det=False)
>   print(res)
>   ```
> * 完整报错/Complete Error Message：
>   ```
>   [[('2中', 0.9981159567832947)]]
>   ```
> 
> 我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no): no
> 
> 请尽量不要包含图片在问题中/Please try to not include the image in the issue. ![missing](https://private-user-images.githubusercontent.com/8534661/268569543-830e8e40-1c61-4e35-ae4e-1058f67c4741.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTIxMjc0MjUsIm5iZiI6MTcxMjEyNzEyNSwicGF0aCI6Ii84NTM0NjYxLzI2ODU2OTU0My04MzBlOGU0MC0xYzYxLTRlMzUtYWU0ZS0xMDU4ZjY3YzQ3NDEuUE5HP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDQwMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA0MDNUMDY1MjA1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZTkwOWFmZTEzYzYyMzhlZjk0NTlkZWU3ZTBiODM3MzBhNmRkMWJlN2E3YTI2NDE3MmUwYWE1ODFhNjY4NmMxMSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.oBKa1dfohPlC3Y2HIpdFOV5F-vAAcTbuUYfzpcVaZuU)

我也遇到了类似的问题，数字中指数无法很好的识别出来。请问后来你是具体如何调整的？
